setup.py

```diff
@@ -29,15 +29,14 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'click==7.0',
+        'click',
```

Comment:
Unless you are sure our code works with any version of click, we should keep the old pinned version as a lower limit.

```suggestion
        'click>=7',
```